soccer/vision/robot/CameraRobot.cpp

```diff
@@ -0,0 +1,49 @@
+#include "CameraRobot.hpp"
+#include <iostream>
+
+RJ::Time CameraRobot::getTimeCaptured() const {
+    return timeCaptured;
+}
+
+Geometry2d::Point CameraRobot::getPos() const {
+    return pos;
+}
+
+double CameraRobot::getTheta() const {
+    return theta;
+}
+
+int CameraRobot::getRobotID() const {
+    return robotID;
+}
+
+CameraRobot CameraRobot::CombineRobots(const std::list<CameraRobot>& robots) {
```

Comment:
If we're combining robot measurements, are we going to do anything about the covariance estimate of the measurement step? Multiple measurements resolving as the same robot should be less certain, _probably_. Also _probably_ a pain to implement (you'd need to keep track of actual covariance instead of just variance everywhere), so it's _probably_ not worth bothering. Just a thought though.

Reply:
Probably not this time. It's there if anyone wants to go back and play with it, but I don't really feel like trying to do the math and figure it out

Reply:
Yeah, makes sense. :+1: